### PR TITLE
Refactor help page content

### LINKED ISSUE #678: Help Page Sentence Structure Refactor
Issue raised by: developer

### What?
The Help pages have a number of grammatical flaws. Go through the help page and amend these.

### Where?
* WebApp/autoreduce_webapp/templates/help.html

### How?
To ensure that the help page is clear and concise for users.

### How to test the issue is resolved
* Proof read help pages to ensure no grammatical errors are present
* Ensure each section of content is clear and concise.

### PR DESCRIPTION
**Pull request by JackEAllen**
### Summary of work
Refactor help pages to fix minor grammatical errors and fix outdated hyperlinks

### How to test your work
Proof read help pages, ensuring all hyperlinks take you to the expected location

Fixes #678

### CODE REVIEW

File: WebApp/autoreduce_webapp/templates/help.html [hunk 1]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
-                To make sure that reductions coming straight from the instrument use new reduce.py scripts automatically, on the All Jobs page
-                click on one of the instrument name, which brings up the Upcoming Configurations page. Then on the left hand side of this page
-                click on 'edit variables' in the Current Variables section on this page. This brings up a Configure New Jobs page. Here ensure
-                that the 'Track script changes' checkbox is ticked.
+                To make sure that reductions coming straight from the instrument use new <code>reduce.py</code>
+                scripts automatically, you can navigate to the instrument summary page fort a
+                given instrument, and select the "See instrument variables" located on the right
+                in relation ot the status table to view ongoing and upcoming instrument
+                configurations.
+                From here, you can select "edit variables" on the left side of the upcoming
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:112]: `"edit variables"` capitalised?

File: WebApp/autoreduce_webapp/templates/help.html [hunk 2]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+                variables table. WHen presented with the variables, ensure that the
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:113]: `When`

File: WebApp/autoreduce_webapp/templates/help.html [hunk 3]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+                "Track script changes" checkbox is checked. this is located on the right side of
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:114]: `This`

File: WebApp/autoreduce_webapp/templates/help.html [hunk 4]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+                the page. You should then select the "Submit Changes" button at the bottom of the
+                page to save this change.
             </p>
             <p>
-                When re-running a job, if you want the re-run to use the current reduce.py script click the 'Reset to current script and values' link under Additional Actions.
+                When re-running a job, if you want the re-run to use the current <code>reduce.py</code> script
+                click the "Reset to current script and values" link under Additional Actions located
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:120]: "Additional Actions"?

File: WebApp/autoreduce_webapp/templates/help.html [hunk 5]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+                on the right of the table.
             </p>
             <p>
-                To update instrument variables to the new ones entered into the reduce_vars.py script go to the Configure New Jobs page and click 'Reset to values in current script', this must be done any time reduce_vars.py is modified.
+                To update instrument variables to the new ones entered into the <code>reduce_vars.py</code>
+                script, navigate to the "Configure New Jobs" page from the "Instrument Summary" Page
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:125]: `page` lowercase

File: WebApp/autoreduce_webapp/templates/help.html [hunk 6]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+                and click "Reset to values in current script", located on the right side of hte page.
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:126]: `the`

File: WebApp/autoreduce_webapp/templates/help.html [hunk 7]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+                This must be done any time <code>reduce_vars.py</code> is modified.
+            </p>
+        </div>
+    </section>
+
+    <section class="help-topic panel panel-default" data-topics="script compatible work">
+        <div class="panel-heading">
+            <h4>How can I make a reduction script compatible with autoreduction?</h4>
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:134]: `Autoreduction` capitalisation

File: WebApp/autoreduce_webapp/templates/help.html [hunk 8]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+        </div>
+        <div class="panel-body">
+            <p>
+                For a reduction script to be compatible with the autoreduction web application
+                it must be named <code>reduce.py</code> expose what variables can be modified.
+                These variables must be made available in a file named
+                <code>reduce_vars.py</code>. For an example of this please See
+                <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Getting-Started">Here</a> for documentation.
+                and <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Example-reduction-script">Here</a>
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:143]: `see` and `here` to lowercase. But I'm not sure if this is intentional as other links are capitalised?

File: WebApp/autoreduce_webapp/templates/help.html [hunk 9]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+                for the full script.
+                Autoreduction expects the reduction script to have a <code>main()</code> method
+                that takes in two arguments, <strong>data</strong> and <strong>output</strong>.
+                These are passed in a kwargs but it may be desirable for these to also be accepted from the command line.
+                For an example of how to do this please
+                <a href="https://github.com/ISISScientificComputing/autoreduce/wiki/Getting-Started#main">See Here</a>.
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:149]: "see [Here]()" with `here` being outside the link like the rest of the links?

File: WebApp/autoreduce_webapp/templates/help.html [hunk 10]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+        </div>
+    </section>
+
+    <section class="help-topic panel panel-default" data-topics="reduction data file location directory output reduced">
+        <div class="panel-heading" id="reduction_data_location">
+            <h4>Where does my reduced data go?</h4>
+        </div>
+        <div class="panel-body">
+            <p>
+                When a reduction job has completed the reduced data is moved to the CEPH storage
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:188]: `When a reduction job has completed, the...`

File: WebApp/autoreduce_webapp/templates/help.html (hunk 1 above)
Comment (JackEAllen) [reply WebApp/autoreduce_webapp/templates/help.html:112]: Leaving this as is to quote link text exactly including capitalisation (or lack of in this case)

File: WebApp/autoreduce_webapp/templates/help.html [hunk 11]
```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
...
+    </section>
+
+    <section class="help-topic panel panel-default" data-topics="script compatible work">
+        <div class="panel-heading">
+            <h4>How can I make a reduction script compatible with autoreduction?</h4>
+        </div>
+        <div class="panel-body">
+            <p>
+                For a reduction script to be compatible with the autoreduction web application
+                it must be named <code>reduce.py</code> expose what variables can be modified.
```
[the remark this answers is not in the document]
Comment (JackEAllen) [reply WebApp/autoreduce_webapp/templates/help.html:139]: Users will understand this - I have modified it slightly

File: WebApp/autoreduce_webapp/templates/help.html (hunk 8 above)
Comment (JackEAllen) [reply WebApp/autoreduce_webapp/templates/help.html:143]: This is intentional 👍

File: WebApp/autoreduce_webapp/templates/help.html (hunk 10 above)
Comment (JackEAllen) [reply WebApp/autoreduce_webapp/templates/help.html:188]: Leaving this one as it's more verbose worded this way.

File: WebApp/autoreduce_webapp/templates/help.html [hunk 12]
```diff
@@ -48,108 +48,155 @@ <h2>ISIS Compute</h2>
...
             <p>
-                When re-running a job, if you want the re-run to use the current reduce.py script click the 'Reset to current script and values' link under Additional Actions.
+                When re-running a job, if you want the re-run to use the current <code>reduce.py</code> script
+                click the "Reset to values in current script" link under "Additional Actions" located
+                on the right side of the page.
             </p>
             <p>
-                To update instrument variables to the new ones entered into the reduce_vars.py script go to the Configure New Jobs page and click 'Reset to values in current script', this must be done any time reduce_vars.py is modified.
+                To update instrument variables to the new ones entered into the <code>reduce_vars.py</code>
+                script, navigate to the  "Instrument Summary" page and select the
```
Comment (rickybassom) [inline WebApp/autoreduce_webapp/templates/help.html:125]: double space before first `"`

**Pull request merged**